src/openforms/prefill/contrib/objects_api/plugin.py

```diff
@@ -79,3 +81,6 @@ def configuration_context(cls) -> JSONObject | None:
                 for group in ObjectsAPIGroupConfig.objects.iterator()
             ]
         }
+
+    def verify_initial_data_ownership(self, submission: Submission) -> None:
+        validate_object_ownership(submission)
```

Comment:
this breaks the isolation between prefill and registration modules because the implementation of `validate_object_ownership` looks at registration backends.

The configuration of which objects API to use is already present in the prefill options, they are passed to `get_prefill_values_from_options` so you need to make sure this also happens when calling `verify_initial_data_ownership`